packages/react-native-reanimated/Common/cpp/reanimated/CSS/common/values/CSSColor.cpp

```diff
@@ -9,6 +9,21 @@ CSSColor::CSSColor()
 CSSColor::CSSColor(ColorType colorType)
     : channels{0, 0, 0, 0}, colorType(colorType) {}
 
+CSSColor::CSSColor(double numberValue)
+    : channels{0, 0, 0, 0}, colorType(ColorType::Rgba) {
+  uint32_t color;
+  if (numberValue < 0) {
+    color = static_cast<int32_t>(numberValue);
```

Comment:
This is implemented incorrectly. There will be narrowing conversion to unsigned 32-bit int which will produce something strange, look:
<img width="1002" height="1658" alt="image" src="https://github.com/user-attachments/assets/138c7e61-487c-4f3b-b4db-39d3edf0653c" />